Docs/source/install/dependencies.rst

```diff
@@ -119,16 +119,24 @@ Without MPI:
 .. code-block:: bash
 
    conda create -n warpx-dev -c conda-forge blaspp ccache cmake compilers git lapackpp openpmd-api python numpy scipy yt fftw matplotlib mamba ninja
-   conda activate warpx-dev
+   source activate warpx-dev
```

Comment:
changed because `source activate` does not pollute `.bashrc`: https://collegeville.github.io/CW20/WorkshopResources/WhitePapers/huebl-working-with-multiple-pkg-mgrs.pdf